config.py

```diff
@@ -90,7 +90,12 @@ class vedaAppSettings(BaseSettings):
 
     veda_stac_root_path: str = Field(
         "",
-        description="Optional path prefix to add to all api endpoints. Used to infer url of stac-api before app synthesis.",
+        description="STAC API root path. Used to infer url of stac-api before app synthesis.",
+    )
+
+    veda_raster_root_path: str = Field(
+        "",
+        description="Raster API root path",
```

Comment:
Is this description the same a `veda_stac_root_path`? Or is this one not inferred

Reply:
I think technically it is inferred since I will be using it to produce the raster api url, but the inferences is not for the use of the stac-api before synthesis.

```
raster_api_url = f"{base_api_url}{veda_app_settings.veda_raster_root_path}/"
```

I can add that bit to the description for the raster root path if you think that's necessary!

Reply:
Nah all good! Was just checking!